src/components/CallToContribute.js

```diff
@@ -61,10 +61,13 @@ const Description = styled.p`
   line-height: 140%;
   color: ${(props) => props.theme.colors.text};
   font-family: "SFMono-Regular", monospace;
+  font-family: "SFMono-Regular", Consolas, "Roboto Mono", "Droid Sans Mono",
+    "Liberation Mono", Menlo, Courier, monospace;
```

Comment:
```suggestion
  font-family: "SFMono-Regular", Consolas, "Roboto Mono", "Droid Sans Mono",
    "Liberation Mono", Menlo, Courier, monospace;
```

![image](https://user-images.githubusercontent.com/54227730/134409369-000df1b3-c974-4c82-ae8d-9b3661253d0c.png)
@minimalsm were these skipped intentionally?

Reply:
Just pushed a patch for this =)